python/lsst/ip/diffim/makeKernel.py

```diff
@@ -119,8 +130,14 @@ def run(self, template, science, kernelSources, preconvolved=False):
                 Spatially varying background-matching function.
         """
         kernelCellSet = self._buildCellSet(template.maskedImage, science.maskedImage, kernelSources)
-        templateFwhmPix = getPsfFwhm(template.psf)
-        scienceFwhmPix = getPsfFwhm(science.psf)
+        templateFwhmPix = evaluateMeanPsfFwhm(template,
+                                              fwhmExposureBuffer=self.config.fwhmExposureBuffer,
+                                              fwhmExposureGrid=self.config.fwhmExposureGrid
+                                              )
+        scienceFwhmPix = evaluateMeanPsfFwhm(science,
+                                             fwhmExposureBuffer=self.config.fwhmExposureBuffer,
+                                             fwhmExposureGrid=self.config.fwhmExposureGrid
+                                             )
```

Comment:
I don't know how I missed this. @yalsayyad reports an increase of 4 seconds in `diffim` and I believe this is all due to these changes that always evaluate the mean FWHM over a 10x10 grid.